views/searchWithFilter.njk

```diff
@@ -135,42 +134,47 @@
         </form>
         </div>
         <div class="moj-filter-layout__content">
-            <div class="govuk-grid-row">
+            <div>
               <div class="govuk-grid-column-full">
               {% if results.length %}
                 {{ mojPagination(pagination) }}
 
                 {% set rows = [] %}
                 {% for u in results %}
-                  {% set userLink = '<a href="' + maintainUrl + '/' + u.username + '/details" class="govuk-link" data-qa="edit-button-'
-                    + u.username +'">' + u.firstName + '&nbsp;' + u.lastName + '</a>' %}
-                {% set tags %}
-                {{ govukTag({
-                  text: "Inactive",
-                  classes: "govuk-tag--pink"
-                }) if not u.active }}
-                {{ govukTag({
-                  text: "Active",
-                  classes: "govuk-tag--green"
-                }) if u.active }}
-                {% endset %}
+                  {% set maintainUserUrl = maintainUrl + '/' + u.username + '/details' %}
+                  {% set cellHtml %}
+                    <div class="govuk-!-margin-bottom-5">
+                      <div class="govuk-grid-row govuk-!-margin-bottom-2">
+                        <div class="govuk-grid-column-full">
+                          <a href="{{maintainUserUrl}}" class="govuk-link" data-qa="edit-button-{{u.username}}">{{u.firstName}}&nbsp;{{u.lastName}}</a><span> - {{u.username}}</span>
+                        </div>
+                      </div>
+                      <div class="govuk-grid-row govuk-!-margin-bottom-2">
+                        <div class="govuk-grid-column-full"><span class="govuk-visually-hidden">Email</span><span>{{u.email}}</span></div>
+                      </div>
+                      <div class="govuk-grid-row govuk-!-margin-bottom-2">
+                        <div class="govuk-grid-column-full">
+                        <span class="govuk-visually-hidden">Status</span>
+                          {{ govukTag({
+                            text: "Inactive",
+                            classes: "govuk-tag--pink"
+                          }) if not u.active }}
+                          {{ govukTag({
+                            text: "Active",
+                            classes: "govuk-tag--green"
+                          }) if u.active }}
+                          <span>at <strong>{{u.activeCaseLoadId}}</strong></span>
+                      </div>
+                      </div>
+                    </div>
+                  {% endset %}
+
```

Comment:
since table is no longer wide enough for a multi-column layout, just show data without columns. TODO add a bit more accessibility for the the hidden labels